docs/SUMMARY.md

```diff
@@ -4,9 +4,6 @@
   - [Local Debug](./local_debug.md)
 - [AggOracle](./aggoracle.md)
 - [Aggsender](./aggsender.md)
-- [Bridge service](./bridge_service.md)
-- [EthTxManager](./ethtxmanager.md)
-- [Etherman](./etherman.md)
```

Comment:
Is this removed by mistake? IMO we should have exposed at least the `bridge_service.md` as the top level page.